morph/react-dom.js

```diff
@@ -31,8 +31,7 @@ export default ({
     animations: {},
     cssDynamic: false,
     cssStatic: false,
-    hasData: !!view.parsed.view.data,
-    data: view.parsed.view.data,
+    hasData: false,
```

Comment:
do we still need these vars?

Reply:
yes, the `hasData` is set inside `block-add-data` step and based on that the `Data.js` is imported into the view.js file. let me know if you think there is a better way to accomplish it

Reply:
if it's only used for a check before calling use, then you could just call use at that point perhaps?

Reply:
got it, I will fix it. thanks